geth/jail/doc.go

```diff
@@ -1,86 +1,91 @@
 /*
-jail - jailed enviroment for executing JS code.
+# jail [![GoDoc](https://godoc.org/github.com/status-im/status-go/geth/jail?status.png)](https://godoc.org/github.com/status-im/status-go/geth/jail)
 
-Package jail implements "jailed" enviroment for executing arbitrary
+jail - jailed environment for executing JS code.
+
+Package jail implements "jailed" environment for executing arbitrary
 JavaScript code using Otto JS interpreter (https://github.com/robertkrimen/otto).
 
 Jail create multiple Cells, one cell per status client chat. Each cell runs own
 Otto virtual machine and lives forever, but that may change in the future.
 
-  +----------------------------------------------+
-  |                     Jail                     |
-  +----------------------------------------------+
-  +---------+ +---------+ +---------+  +---------+
-  |  Cell   | |  Cell   | |  Cell   |  |  Cell   |
-  |ChatID 1 | |ChatID 2 | |ChatID 3 |  |ChatID N |
-  |+-------+| |+-------+| |+-------+|  |+-------+|
-  ||Otto VM|| ||Otto VM|| ||Otto VM||  ||Otto VM||
-  |+-------+| |+-------+| |+-------+|  |+-------+|
-  || Loop  || || Loop  || || Loop  ||  || Loop  ||
-  ++-------++ ++-------++ ++-------++  ++-------++
-
-Cells
-
+```
```

Comment:
So here is the problem: godoc is not in Markdown. It uses indentation for code blocks, not the ``` clauses.